packages/@aws-cdk/aws-lambda/test/integ.lambda.ts

```diff
@@ -31,4 +32,8 @@ alias.addFunctionUrl({
   authType: lambda.FunctionUrlAuthType.NONE,
 });
 
+// Changes the function description when the feature flag is present
+// to validate the changed function hash.
+cdk.Aspects.of(stack).add(new lambda.FunctionVersionUpgrade(LAMBDA_RECOGNIZE_LAYER_VERSION));
```

Comment:
Added this aspect here, so that `yarn integ --update-on-failed` succeeds. For the integ test updates in other modules, I just ran `yarn integ --update-on-failed --disable-update-workflow --dry-run`.